src/commands/flutter-symbols/__tests__/fixtures/pubspecs/missingVersionPubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: valid_pubspec_example
+description: A pubspec that's valid
```

Comment:
Is this pubspec still valid even if the version is missing?

Reply:
Flutter doesn't complain about it, though to be honest I have no idea what it does in that case? Likely it just doesn't modify the versions set in platform specific files.